backend/lcfs/web/api/transfer/views.py

```diff
@@ -19,17 +23,20 @@ async def create_transfer(transfer: schema.TransferBase, db: AsyncSession = Depe
     except Exception as e:
         raise HTTPException(status_code=500, detail=f"Internal Server Error: {str(e)}")
 
-@router.get("/transfers/", response_model=List[schema.Transfer])
+
+@router.get("/", response_model=List[schema.Transfer])
```

Comment:
im not sure, I dont think these were working when jignesh created these, it may have just been scaffolding. It would be good to tackle this while doing the transfers/transactions page tasks